src/seedu/addressbook/data/person/address/Address.java

```diff
@@ -27,13 +39,22 @@ public Address(String address, boolean isPrivate) throws IllegalValueException {
             throw new IllegalValueException(MESSAGE_ADDRESS_CONSTRAINTS);
         }
         this.value = trimmedAddress;
```

Comment:
No need of this value anymore. It is a duplicate information and can lead to inconsistent data if you use it in the toString()  method.